actors.md

```diff
@@ -477,6 +478,16 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 		miner.ProvingPeriodEnd = chain.Now() + ProvingPeriodDuration(miner.SectorSize)
 	}
 }
+
+func collateralForPower(power BytesAmount) TokenAmount {
+  availableFil = FakeGlobalMethods.GetAvailableFil()
+  totalNetworkPower = StorageMinerActor.GetTotalStorage()
+  numMiners = FakeGlobalMethods.GetTotalNumMiners()
```

Comment:
Suggest `StorageMarket.GetMinerCount()` (which implementation is pretty obvious for us to follow-up with).

Reply:
Done.